Sources/NIOWebSocketServer/main.swift

```diff
@@ -225,10 +225,19 @@ defer {
 }
 
 
-let channel = try! bootstrap.bind(host: "localhost", port: 8888).wait()
-print("Server started and listening on \(channel.localAddress!)")
+do {
+    let channel = try bootstrap.bind(host: "localhost", port: 8888).wait()
 
-// This will never unblock as we don't close the ServerChannel
-try channel.closeFuture.wait()
+    if let localAddress = channel.localAddress {
+        print("Server started and listening on \(localAddress)")
+    } else {
+        print("Server started but the address could not be determined.")
+    }
+    
+    // This will never unblock as we don't close the ServerChannel
+    try channel.closeFuture.wait()
```

Comment:
@agnosticdev sorry, would you mind taking this line out just like for the other servers?

Reply:
@weissi thank you for the feedback, this PR has now been updated.